### PR TITLE
[RUM-3349] RUM Browser consent management doc

### PR DESCRIPTION
**Pull request by BenoitZugmeyer**
### What does this PR do? What is the motivation?

Document https://github.com/DataDog/browser-sdk/pull/2634

This is inspired by the RUM Mobile documentation: https://docs.datadoghq.com/real_user_monitoring/mobile_and_tv_monitoring/troubleshooting/#set-tracking-consent-gdpr-compliance

### Merge instructions

<s>This should be merged after https://github.com/DataDog/browser-sdk/pull/2634 is released. Please ask me or @nhientrandatadog  before merging.</s>

The feature has been released, please merge after review!

### CODE REVIEW

**Comment by cswatt** [comment]
created DOCS-7391 to review.

**Review by bcaudan — APPROVED** [review]
LGTM

**Review by nhientrandatadog — APPROVED** [review]
lgtm!

File: content/en/real_user_monitoring/browser/_index.md [hunk 1]
```diff
@@ -1730,6 +1730,12 @@ The application's environment, for example: prod, pre-prod, and staging. Follows
 **Type**: String<br/>
 The application's version, for example: 1.2.3, 6c44da20, and 2020.02.13. Follows the [tag syntax requirements][15].
 
+`trackingConsent`
+: Optional<br/>
+**Type**: `"granted"` or `"not-granted"`<br/>
+**Default**: `"granted"`<br/>
+Whether the user granted their tracking consent. See [User Tracking Consent][27].
```
Comment (rtrieu) [inline content/en/real_user_monitoring/browser/_index.md:1737]:
```suggestion
Affirm whether the user has granted their tracking consent. See [User Tracking Consent][27].
```

File: content/en/real_user_monitoring/browser/advanced_configuration.md [hunk 2]
```diff
@@ -734,6 +734,69 @@ window.DD_RUM &&
 
 For a sampled out session, all page views and associated telemetry for that session are not collected.
 
+## User tracking consent
+
+To be compliant with the GDPR, CCPA and similar regulations, the RUM Browser SDK allows to provide the tracking consent value at initialization.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/browser/advanced_configuration.md:739]:
```suggestion
To be compliant with the GDPR, CCPA and similar regulations, the RUM Browser SDK lets you provide the tracking consent value at initialization.
```

File: content/en/real_user_monitoring/browser/advanced_configuration.md [hunk 3]
```diff
@@ -734,6 +734,69 @@ window.DD_RUM &&
...
+
+The `trackingConsent` initialization parameter can be one of the following values:
+
+1. `"granted"`: The RUM Browser SDK starts collecting data and sends it to Datadog.
+2. `"not-granted"`: The RUM Browser SDK does not collect any data.
+
+To change the tracking consent value after the RUM Browser SDK is initialized, use the `setTrackingConsent()` API call. The RUM Browser SDK changes its behavior according to the new value:
+
+* when changed from `"granted"` to `"not-granted"`, the RUM session is stopped, and no data will be sent anymore.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/browser/advanced_configuration.md:748]:
```suggestion
* when changed from `"granted"` to `"not-granted"`, the RUM session is stopped, data is no longer sent to Datadog.
```

File: content/en/real_user_monitoring/browser/advanced_configuration.md [hunk 4]
```diff
@@ -734,6 +734,69 @@ window.DD_RUM &&
...
+* when changed from `"not-granted"` to `"granted"`, a new RUM session is be created if no previous session is active, and data collection is be started again.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/browser/advanced_configuration.md:749]:
```suggestion
* when changed from `"not-granted"` to `"granted"`, a new RUM session is created if no previous session is active, and data collection resumes.
```

File: content/en/logs/log_collection/javascript.md [hunk 5]
```diff
@@ -359,6 +359,7 @@ The following parameters are available to configure the Datadog browser logs SDK
...
 | `sampleRate`               | Number                                                                    | No       | `100`           | **Deprecated** - see `sessionSampleRate`.                                                                                                                                             |
 | `sessionSampleRate`        | Number                                                                    | No       | `100`           | The percentage of sessions to track: `100` for all, `0` for none. Only tracked sessions send logs.                                                                                    |
+| `trackingConsent`          | `"granted"` or `"not-granted"`                                            | No       | `"granted"`     | Whether the user granted their tracking consent. See [User Tracking Consent][15].                                                                                                     |
```
[the remark this answers is not in the document]
Comment (rtrieu) [reply content/en/logs/log_collection/javascript.md:362]: Not sure if this is an affirmation or indication of the user's preference.

**Review by rtrieu — CHANGES_REQUESTED** [review]
Some minor feedback suggestions, but otherwise looks good!

File: content/en/logs/log_collection/javascript.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (BenoitZugmeyer) [reply content/en/logs/log_collection/javascript.md:362]: I'm not sure about what's the best wording here. I'm not used to see the verb "affirm" within documentation... maybe we could keep it simple and reword it as:
```
Set the initial user tracking consent state.
````
so it's in-line with the `setTrackingConsent` API wording

[the remark this answers is not in the document]
Comment (rtrieu) [reply content/en/logs/log_collection/javascript.md:362]: Thanks Benoit, I agree with you, I think set would be the most simple and clear here.

File: content/en/real_user_monitoring/browser/advanced_configuration.md [hunk 6]
```diff
@@ -734,6 +734,69 @@ window.DD_RUM &&
 
 For a sampled out session, all page views and associated telemetry for that session are not collected.
 
+## User tracking consent
+
+To be compliant with GDPR, CCPA and similar regulations, the RUM Browser SDK lets you provide the tracking consent value at initialization.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/browser/advanced_configuration.md:739]:
```suggestion
To be compliant with GDPR, CCPA, and similar regulations, the RUM Browser SDK lets you provide the tracking consent value at initialization.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Hi @BenoitZugmeyer, sorry, I missed one edit.

**Review by rtrieu — APPROVED** [review]
Looks good, thanks!

**Pull request merged**